flink-table/flink-table-planner/src/main/java/org/apache/flink/table/planner/plan/schema/CatalogSourceTable.java

```diff
@@ -119,7 +119,7 @@ public RelNode toRel(ToRelContext toRelContext) {
                 !schemaTable.isStreamingMode(),
                 context.getTableConfig(),
                 relBuilder,
-                schemaTable.getContextResolvedTable(),
+                catalogTable,
```

Comment:
Neat.  So the resolved table does not include the hints?

Does the output of `computeContextResolvedTable` contain anything else of interest?  (AKA, are we going to come back for something like OPTIONS later?)

Reply:
>So the resolved table does not include the hints?

in fact it didn't include hints before line 108 in this class https://github.com/apache/flink/blob/7f0e0e81bac2cc512d0669b25995ca3b13e262f9/flink-table/flink-table-planner/src/main/java/org/apache/flink/table/planner/plan/schema/CatalogSourceTable.java#L108-L111

where then hints (including option hint) are going to be merged into `catalogTable`, however for some reason it was not used here